### DISCUSSION TITLE
Display ILI9341 wiring 6 vs 9 pins

### DISCUSSION THREAD

**Discussion by tasmotari** [arendst/Tasmota#17314] [General]
Hello!

Display ILI9341 need to wiring 9 pins for ESP (ideally): 
**MISO, MOSI, CLK, CS, DC, RST, LED, VCC, GND**

But it also works fine without **MISO**.
**LED** and **RST** can be wired to the 3.3V regulator on display board. 
Than you need only 6 pins.

How reliable is it?

**Comment by sfromis** [comment]
Ideally, you also have the touch CS pin. WIthout LED, you lose control over backlight. Without MISO, you lose the ability to use touch. Not sure what the effects are of not having reset.
